<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>people</class>
 <widget class="QWidget" name="people">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>274</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolButton" name="toolBut_name">
      <property name="maximumSize">
       <size>
        <width>39</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>昵称:</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolBut_birthday">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>生日:             请输入格式为：2002-09-10      </string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_birthday"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolBut_sex">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>性别:             请输入格式为：男/女                </string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_sex"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolBut_cons">
      <property name="maximumSize">
       <size>
        <width>39</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>星座:</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_cons"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolBut_intr">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>自我介绍:</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_intr">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushBut_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
